versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Installing Rancher Prime PAYG on Azure
+---
+
+This page covers installing the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.
+
+## How to Install Rancher Prime PAYG
+
+The following steps describe how to create a new deployment of Rancher Prime from the [Azure Marketplace](https://azuremarketplace.microsoft.com) page.
+
+1. Select the **Rancher Prime with 24x7 Support** offer (either **EU and UK only** or **non-EU and non-UK only**) that corresponds to the location where your account is registered.
+1. Choose a plan from the dropdown menu. View the **Plans + Pricing** tab for more details about the plan.
+1. Select **Create**.
+
+### Basics
+
+On the **Basics** tab, specify the **Project details** and **Instance details**:
+
+![Basics tab](/img/install-rancher-prime-basics.png)
+
+1. Select an existing **Subscription** from the dropdown menu.
+1. Select an existing **Resource group** from the dropdown menu.
+
+   :::note
+
+   The **Create new** resource group feature is not supported.
+
+   ![Create new resource group not supported](/img/install-rancher-prime-basics-create-new.png)
+
+   :::
+
+1. Select an existing **AKS Cluster Name** from the dropdown menu.
+1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots and must be between 2 and 253 characters long.
+1. Select **Next**.
+
+### Rancher Configuration
+
+On the **Rancher Configuraion** tab, specify the following information:
+
+![Rancher Configuration](/img/install-rancher-prime-bootstrap-password.png)
+
+1. Enter the **Hostname** for Rancher. The Rancher hostname must be a fully qualified domain name (FQDN). The Rancher server URL will be created using this hostname.
+   :::note
+
+   The IP address of the Rancher hostname must be resolvable by a public DNS.
+
+   :::
+1. Using the slider, select the number of **Replicas**.
+1. Choose and confirm a **Bootstrap Password**. During the first login, you will use the bootstrap password to authenticate to the Rancher dashboard.
+   :::note
+
+   The current Rancher deployment exposes the bootstrap password in the Cluster configuration settings in the Azure Portal. Until this security issue is resolved, we suggest changing the Admin password after initial login, by editing your profile in the Rancher dashboard.
+
+   :::
+1. Select **Next**.
+
+### Review + Create
```

Comment:
```suggestion
### Review + create
```

I think the initial commit with the lowercase form is correct because this header (and the previous 2) are based off the actual tab label used in the UI. This aligns with the screenshot and line 59.